docs/how_to_contribute.md

```diff
@@ -192,7 +188,7 @@ Thus `https://www.latimes.com/news-sitemap.xml` is a Google News Index Map.
 
 #### Finishing the Publisher Specification
 
-1. Sometimes spanning the entire site means spanning the entire site, so sitemaps can include a lot of noise like sitemaps pointing to a collection of tags or authors, etc.
+1. Sometimes spanning the entire site means the corresponding sitemaps can include a lot of noise like maps pointing to a collection of tags or authors, etc.
```

Comment:
This sentence does not quite work out. How about: 
"Some sitemaps contain content that does not represent articles. These types of sitemap are not useful for extracting links to articles. Furthermore, these sitemaps might contain duplicate links. It is recommended to ignore these kinds of sitemaps "

Reply:
I would go with `Sometimes sitemaps can include a lot of noise like maps pointing to a collection of tags or authors, etc.`
The duplication part isn't important here.

Reply:
Alright, looks good to me.